### ISSUE TITLE
SPAv3 Support in phpFinTS

### ISSUE THREAD

**Issue by ssnailed** [bnw/firefly-iii-fints-importer#137]
Hello, a few weeks ago the phpFinTS library [implemented support](https://github.com/nemiah/phpFinTS/pull/432) for SPAv3, which would allow this importer to work with the Baader Bank (custodian bank for some neobrokers like finanzen.net zero, scalable.capital). Would it be possible to update to this new version?

**Comment by se7entynine** [comment]
This project is not supported anymore. Check the alternative variants to import your data, e.g. GoCardless or Salt edge or the old way with a csv file.

**Comment by bnw** [comment]
Yes, unfortunately, I don't have a lot of time for this project. 
If you have a bit of a technical background, I encourage you to try out to implement this yourself. I'm happy to review a PR and give support where I can. 
Judging from how small the PR you link is, maybe it is just a simple matter of updating the `nemiah/php-fints` version?

**Comment by ssnailed** [comment]
I'll take a look at it over the weekend if I have some time, I don't expect it to be very complex.
